src/main/resources/META-INF/plugin.xml

```diff
@@ -285,6 +288,12 @@
                     <override-text place="GoToAction" text="Cody: Dismiss"/>
                 </action>
 
+                <action id="cody.inlineEditEditCode"
+                        text="Edit Code"
```

Comment:
I think text should not be `Edit Code`. We already have other `Edit Code` and I think both will be displayed in the key bindings settings with the same name.

Reply:
We have `Edit Code...` action (not the three dots!) - it's a common practice in JB that the actions that open dialogs or need some confirmation have `...` suffix. IMHO it's a good enough to distinguish them (we clearly indicate what the shortcuts do and display them - it's super clear).